mangaki/mangaki/management/commands/tokens.py

```diff
@@ -16,9 +24,27 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         username = options.get('username')[0]
         salt = options.get('salt')[0]
-        if username == '*':
-            for user in get_user_model().objects.filter(profile__newsletter_ok=True):
-                if user.email:
-                    pass  # Send a mail
-        else:
+        newsletter = yaml.load(open(os.path.join(settings.DATA_DIR, 'newsletter.yaml')).read())
```

Comment:
Add a `os.access` to see if the file exists / can be accessed and throw an error / inform the user in case where `newsletter.yaml` is not available.